### PR TITLE
update to bevy 0.4

### LINKED ISSUE #6: request: update to bevy 0.4
https://github.com/bevyengine/bevy/blob/v0.4.0/CHANGELOG.md

### PR DESCRIPTION
**Pull request by phrohdoh**
```
$ git rev-parse HEAD
05ff172322e95b6665e39dfe2ccd1c9f670785fa

$ git branch --show-current
bevy-0.4

$ cargo t --all
    Finished test [unoptimized + debuginfo] target(s) in 0.12s
     Running target/debug/deps/bevy_svg_map-6c1e89de9eec9256

running 3 tests
test style::tests::style_parse_default ... ok
test style::tests::test_stroke_width ... ok
test tests::tokenize_properly ... ok

test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/integration_test-69521b9387d1ba9b

running 4 tests
test custom_style_strategy ... ok
test can_it_be_added ... ok
test whole_svg_with_shapes ... ok
test whole_svg ... ok

test result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

   Doc-tests bevy_svg_map

running 6 tests
test src/style.rs - style::SvgStyle::stroke_linecap (line 148) ... ok
test src/style.rs - style::SvgStyle::stroke_opacity (line 116) ... ok
test src/style.rs - style::SvgStyle::stroke_dasharray (line 101) ... ok
test src/style.rs - style::SvgStyle::stroke_dasharray (line 89) ... ok
test src/style.rs - style::SvgStyle (line 54) ... ok
test src/style.rs - style::SvgStyle::stroke_linejoin (line 173) ... ok

test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```

if merged this will close #6

### CODE REVIEW

**Comment by phrohdoh** [comment]
I _suspect_ the examples in the readme could do with some fixing up, though I do not know the details of what should change.

**Comment by carrascomj** [comment]
Hello, thank you for the update! The PR looks structurally good, I'll check if it works today and come back to this.

File: README.md [hunk 1]
```diff
@@ -111,7 +110,7 @@ Plug and play!
 use bevy_svg_map::load_svg;
 
 fn setup_whole_svg(
-    commands: Commands,
+    commands: &mut Commands,
```
Comment (carrascomj) [inline README.md:113]:
```suggestion
    mut commands: &mut Commands,
```
And inside the function 
```rust
load_svg(&mut commands, materials, meshes, "assets/ex.svg", 1., 2.);
```
That's how it worked for me.

File: README.md [hunk 2]
```diff
@@ -38,14 +38,13 @@ struct MyStrategy;
...
 fn main() {
-    App::build().add_default_plugins().add_startup_system(setup.system()).run();
     App::build()
           .add_default_plugins()
           .add_startup_system(setup.system())
           .run();
 }
 
-fn setup(com: Commands, mat: ResMut<Assets<ColorMaterial>>, mesh: ResMut<Assets<Mesh>>) {
+fn setup(com: &mut Commands, mat: ResMut<Assets<ColorMaterial>>, mesh: ResMut<Assets<Mesh>>) {
```
Comment (carrascomj) [inline README.md:47]:
```suggestion
fn setup(mut com: &mut Commands, mat: ResMut<Assets<ColorMaterial>>, mesh: ResMut<Assets<Mesh>>) {
```

File: README.md [hunk 3]
```diff
@@ -38,14 +38,13 @@ struct MyStrategy;
...
     load_svg_map(com, mat, mesh, "assets/ex.svg", MyStrategy);
```
Comment (carrascomj) [inline README.md:48]:
```suggestion
    load_svg_map(&mut com, mat, mesh, "assets/ex.svg", MyStrategy);
```

**Review by carrascomj — CHANGES_REQUESTED** [review]
LGTM. The **README** has to be updated with some `mut`s, but it is fine. I will upload an example to generate the README image after this.

**Comment by phrohdoh** [comment]
I believe we specifically do not want `mut commands: &mut Commands`, just `commands: &mut Commands`: https://bevyengine.org/news/bevy-0-4/#ecs-improvements

**Comment by carrascomj** [comment]
Oh, my bad, the example that I was using was wrong. Yes, the README is fine as you have it.

**Review by carrascomj — APPROVED** [review]

**Pull request merged**